### ISSUE TITLE
IDE0032 suggestion on improper case

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/roslyn#33460]
Version: 15.9 

In the following code, VS throws an IDE0032 suggestion:

```csharp
        private (double X, double Y) position;

        public (double X, double Y) Position
        {
            get => position;
            set => position = value;
        }
        public double X
        {
            get => position.X;
            set => position.X = value;
        }
        public double Y
        {
            get => position.Y;
            set => position.Y = value;
        }
```

Converting this to an auto property results in a compiler error, which makes sense, since properties' members may not be changed.

_This issue has been moved from https://developercommunity.visualstudio.com/content/problem/436930/ide0032-suggestion-on-improper-case.html
VSTS ticketId: 778746_
_These are the original issue comments:_
(no comments)
_These are the original issue solutions:_
(no solutions)

**Comment by sharwell** [comment]
Duplicate of #28511

**Comment by sharwell** [comment]
📝 This was fixed for 16.0 Preview 1
